target/linux/lantiq/Makefile

```diff
@@ -9,7 +9,7 @@ BOARDNAME:=Lantiq
 FEATURES:=squashfs
```

Comment:
Isn't it time to add ubifs to the mix? 
This should allow fro the configuration of a ubifs image.

FEATURES:=squashfs ubifs

Reply:
This seems completely unrelated here?